src/components/tooltip/README.md

```diff
@@ -0,0 +1,67 @@
+# Tooltip
+
+#### Description
+
+Tooltips display informative text when users hover over or focus on an element.
+
+#### Usage
+
+```js
+<Tooltip
+  position="left"
+  label="If you buy a pizza, you will also get a free ice cream :)"
+>
+  <button onClick={orderPizza({ freeIceCream: 'yes please' })}>Submit</button>
+</Tooltip>
+```
+
+#### Working with disabled child elements
+
+When you use a tooltip with a disabled element, you need to wrap element in a div, as
+disabled elements do not fire events.
+
+```js
+<Tooltip
+  position="left"
+  label="You do not have permission to delete the database"
+>
+  <div style={{ cursor: 'not-allowed' }}>
+    <button
+      disabled
+      onClick={deleteDb()}
+      style={{
+        pointerEvents: 'none',
+      }}
+    >
+      Delete production database
+    </button>
+  </div>
+</Tooltip>
+```
+
+#### Working with custom child elements
+
+The tooltip needs to apply DOM event listeners to its child element. If the child is a custom React element, you need to make sure that it spreads its properties to the underlying DOM element.
```

Comment:
It would be helpful to list the DOM event listeners that we are referring to (`onMouseOver`, etc)